manifests/upstream/calico/crd/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+bases:
```

Comment:
I think this is a little weird to install calico from cloud-provider-gcp, I would add a comment as to why

Reply:
I couldn't find versioned static manifests published by Calico themselves - The calico docs point to unversioned artifacts https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart and the calico github only has the inputs to generate the artefacts rather than the artefacts themselves.  If there's a place where they are actually available that I missed please let me know.

Reply:
Thats fine - Just put that message as a comment in the code